### DISCUSSION TITLE
Nextcord iinstalled but not importing

### DISCUSSION THREAD

**Discussion by ghost** [nextcord/nextcord#584] [Q&A] [answered]
I've installed nextcord on pip, but when I try to import it using: from nextcord.ext import ipc import nextcord it says missing installation. I have the folders in my site-packages folder. I don't know what I am doing wrong

**Comment by DenverCoder1** [comment, marked as answer]
Have you installed [`nextcord-ext-ipc`](https://github.com/nextcord/nextcord-ext-ipc) also?

**Comment by ghost** [comment]
> Have you installed [`nextcord-ext-ipc`](https://github.com/nextcord/nextcord-ext-ipc) also?

It successfully installed it but it is still saying missing imports

**Comment by DenverCoder1** [comment]
There are many potential problems:

1. You're IDE's language server is out of date. Restarting the IDE should fix that issue.

2. You have multiple Python installations and the installation you installed `nextcord-ext-ipc` on is different from the one you're using for running your code

3. You have a virtual enviroment enabled, and `nextcord-ext-ipc` is installed outside of the virtual enviroment.

4. You installed `nextcord-ext-ipc` in a virtual environment, but your virtual environment is not activated.

**Comment by ghost** [comment]
> There are many potential problems:
> 
> 1. You're IDE's language server is out of date. Restarting the IDE should fix that issue.
> 2. You have multiple Python installations and the installation you installed `nextcord-ext-ipc` on is different from the one you're using for running your code
> 3. You have a virtual enviroment enabled, and `nextcord-ext-ipc` is installed outside of the virtual enviroment.
> 4. You installed `nextcord-ext-ipc` in a virtual environment, but your virtual environment is not activated.

I've restarted my IDE multiple times, and I'm not sure how to fix any of the other issues.

I even updated my python

**Comment by ooliver1** [comment]
Alright, what is saying the error, your IDE or when running the file?
What IDE/editor do you use?
How do you run your bot?

**Reply by ghost** [reply]
> Alright, what is saying the error, your IDE or when running the file?
> What IDE/editor do you use?
> How do you run your bot?

The IDE is saying the error. 
I use vs code
I’m not actually running the bot yet it just says that in the code. 

Also I tried responding in my email to this but it may not have worked so I’m trying it here

**Reply by ghost** [reply]
> Alright, what is saying the error, your IDE or when running the file?
> What IDE/editor do you use?
> How do you run your bot?

I ask you again. I am unable to run my program without this package. Sorry if that came across angry though. I didn’t mean it to if It did

**Reply by ooliver1** [reply]
did you make sure your interpreter selected is the correct one (`ctrl + shift + p`; `python: select interpreter`)
